itez/beneficiary/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
+from . import views
 
 
 app_name = "beneficiary"
 urlpatterns = [
-    
+    # path('swagger/', views.schema_view, name='swagger'),
```

Comment:
lets remove this until when we are setting it up

Reply:
Noted